crates/board/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Major
 
+- Change `Id::new` to return `OutOfBounds` error instead of `InvalidArgument`
```

Comment:
I would consider this a patch change because we don't document in the public API which error is returned.

Also nit: let's use parentheses for functions `s/Id::new/Id::new()/` (we can't fix the one from the previous PR otherwise it will mess up `git blame` which I use to generate the release changelog).

Reply:
Done. Sorry about the previous one!

Reply:
No worries, that's not a big deal and I'm the one who missed it.